docs/content/en/docs/getting-started/nutanix/nutanix-spec.md

```diff
@@ -244,9 +244,11 @@ Type to identify the OS image. (Permitted values: `name` or `uuid`)
  
 ### image.name (`name` or `UUID` required)
 Name of the image
+The `image.name` name must have a cluster kubernetes version or workerNodeGroupConfiguration.kubernetesVersion version(in case of modular upgrade) in the name. If the kubernetes version is 1.24, the `image.name` field name should include 1.24, 1_24, 1-24 or 124.
  
 ### image.uuid (`name` or `UUID` required)
 UUID of the image
+The image name of the `image.uuid` field must have a cluster kubernetes version or workerNodeGroupConfiguration.kubernetesVersion version(in case of modular upgrade) in the name. If the kubernetes version is 1.24, the image name of the `image.uuid` field name should include 1.24, 1_24, 1-24 or 124.
```

Comment:
The image name being fetched from uuid. I have tweaked my sentence a bit.

Reply:
I see.

```suggestion
The image name of the `image.uuid` field must have a cluster kubernetes version or workerNodeGroupConfiguration.kubernetesVersion version(in case of modular upgrade) in the name. If the kubernetes version is 1.24, the image name of the `image.uuid` field name should include 1.24, 1_24, 1-24 or 124.

The name of the image associated with the `uuid` must contain the `Cluster.Spec.KubernetesVersion` or `Cluster.Spec.WorkerNodeGroupConfiguration[].KubernetesVersion` version (in case of modular upgrade). For example, if the Kubernetes version is 1.24, `image.name` must include 1.24, 1_24, 1-24 or 124.
```